### PR TITLE
x11-themes/mate-backgrounds: unkeyword

### PR DESCRIPTION
**Pull request by immolo**

### CODE REVIEW

File: x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild [hunk 1]
```diff
@@ -5,8 +5,9 @@ EAPI=8
 
 inherit mate
 
-if [[ ${PV} != 9999 ]]; then
-	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+MINOR=$((ver_cut 2) % 2)
+if [[ ${MINOR} -eq 0 ]]; then
+        KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
```
Comment (juippis) [inline x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild:10]: This seems a bit weird but let's fix the tree first!

**Review by juippis — APPROVED** [review]

**Comment by juippis** [comment]
I'm still getting
```
x11-themes/mate-backgrounds
  MissingInherits: version 1.27.1: eapi7-ver: missing inherit usage: 'ver_cut', line 8
  NonsolvableDepsInDev: version 1.27.1: nonsolvable depset(bdepend) keyword(~amd64) dev profile (default/linux/amd64/17.0/musl) (70 total): solutions: [ >=mate-base/mate-common-1.27 ]
  NonsolvableDepsInStable: version 1.27.1: nonsolvable depset(bdepend) keyword(~amd64) stable profile (default/linux/amd64/17.1) (61 total): solutions: [ >=mate-base/mate-common-1.27 ]
  WhitespaceFound: version 1.27.1: ebuild has leading whitespace on on line: 10
```

**Comment by juippis** [comment]
5d3106e1b8df4c209a8aab2ff7b9f376b0102f9d 

fast fix but this is kind of wrong, we shouldn't define KEYWORDS twice in an ebuild. 
https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0105

we (you ;)) should figure out a better way to handle this...

**Comment by immolo** [comment]
> [5d3106e](https://github.com/gentoo/gentoo/commit/5d3106e1b8df4c209a8aab2ff7b9f376b0102f9d)
> 
> fast fix but this is kind of wrong, we shouldn't define KEYWORDS twice in an ebuild. https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0105
> 
> we (you ;)) should figure out a better way to handle this...

Thanks @juippis a clean up commit is incoming to resolve all of those we introduced ;)

File: x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild [hunk 2]
```diff
@@ -5,8 +5,9 @@ EAPI=8
 
 inherit mate
 
-if [[ ${PV} != 9999 ]]; then
-	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+MINOR=$((ver_cut 2) % 2)
```
Comment (thesamesam) [inline x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild:8]: This is still missing a $.

File: x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild (hunk 1 above)
Comment (thesamesam) [reply x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild:10]: It's actually not very weird - I think we have other examples somewhere. Even releases are stable, odd are development.

File: x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild (hunk 2 above)
Comment (immolo) [reply x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild:8]: Sorry about that, it's fixed in the clean up commit.

**Pull request closed without merging**